composer.lock

```diff
@@ -6557,31 +6558,31 @@
         },
         {
             "name": "twig/twig",
-            "version": "v1.38.4",
+            "version": "v1.42.4",
```

Comment:
Seems like a good thing but we'll need to perform a full check

Reply:
Yes, change of twig version was needed to upgrade the symfony version, but the test failure origin is not much clearer...

Reply:
because of this: https://github.com/PrestaShop/PrestaShop/pull/17288/files#diff-1da2c7edc898c70e5a79a9997c98ceccR6324

Reply:
@PierreRambaud Of course, but how to solve it? Or change the target from 1.7.6.x to 1.7.7.x which should be released soon?

Reply:
Does not look like an issue to me 😄 

I checked twig changelog, nothing suspicious in this update, I think we can do it